tests/pebble.rs

```diff
@@ -431,6 +410,35 @@ impl PebbleEnvironmentConfig {
     }
 }
 
+fn verify_cert(
```

Comment:
Should this be a method on the `pebble`, and internalize the `issuer_roots().await?` call?

Reply:
Done.